speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/mainscreens/DatabaseSettingsMenu.cpp

```diff
@@ -280,16 +160,16 @@ static void SelectCert(int p_buttonControl, int p_labelControl, char* p_normalTe
 
     // Validate input w.r.t. black boxes
     std::experimental::filesystem::path path = buf;
-    // Minimum file length: "{Drive Letter}:\{empty file name}.exe"
+    // Minimum file length: "{Drive Letter}:\{empty file name}.pem"
     if (path.string().size() <= 7)
     {
-        GfuiLabelSetText(s_scrHandle, p_labelControl, MSG_NO_CERT_FILE);
+        GfuiLabelSetText(s_scrHandle, p_labelControl, "please select an appropriate filename");
```

Comment:
Why was this replaced with "hardcoded" string instead of a define?

Reply:
Not sure why this was removed I readded it